packages/vitest/src/types/worker.ts

```diff
@@ -26,7 +27,7 @@ export interface ContextRPC {
   workerId: number
   config: SerializedConfig
   projectName: string
-  files: string[]
+  files: string[] | FileSpec[]
   environment: ContextTestEnvironment
```

Comment:
I wonder, wouldn't it be better if we used `SerializedTestSpecification` for this and added an additional `specs` field instead for better backwards compatibility?

Reply:
Sounds like it, but I'm not sure.  I looked it up in the codebase, it seems to be a bit away from the files I've dealt with.

Seems better to me to use a type we already have (which also seems to be doing what we need).  But I'll have to do some mess around a bit so that I'll be able to access it in `interpretTaskModes`.

The call is yours.

Reply:
Ok, let's just keep it like you already did